test/extended/security/service_serving_cert_signer.go

```diff
@@ -0,0 +1,167 @@
+package security
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	kapiv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	certutil "k8s.io/client-go/util/cert"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+const (
+	serviceName               = "test-service"
+	secretName                = "test-secret"
+	configMapName             = "test-config"
+	servingCertAnnotation     = "service.alpha.openshift.io/serving-cert-secret-name"
+	configMapBundleAnnotation = "service.alpha.openshift.io/inject-cabundle"
+	configMapBundleDataKey    = "service-ca.crt"
+	signerCertNamespace       = "openshift-service-cert-signer"
+	signerCertSecretName      = "service-serving-cert-signer-signing-key"
+)
+
+var _ = g.Describe("[Feature:ServiceServingCert][Conformance][security] service serving certs", func() {
```

Comment:
Remove [Conformance], it is no longer necessary.  I would also remove [security], no real reason to have it.  You want the describe to set the context, so I would probably have the top level be:

> [Feature:ServiceServingCert] Services that request serving certificates

and then remove the describe below and instead have just the it with:

> g.It("see a created secret and config map", ...

instead of having three levels.

Reply:
At that point you get the nice `[Feature:ServiceServingCert] Services that request serving certificates see a created secret and config map`